scheduler/src/cook/kubernetes/api.clj

```diff
@@ -625,9 +625,10 @@
     (.putRequestsItem resources "memory" (double->quantity (* memory-multiplier mem)))
     (.putLimitsItem resources "memory" (double->quantity (* memory-multiplier mem)))
     (.putRequestsItem resources "cpu" (double->quantity cpus))
-    ; We would like to make this burstable, but for various reasons currently need pods to run in the
-    ; "Guaranteed" QoS which requires limits for both memory and cpu.
-    (.putLimitsItem resources "cpu" (double->quantity cpus))
+    (when (:set-container-cpu-limit? (config/kubernetes))
```

Comment:
Can we destructure this on line 574, in the same place as all the other values we pull from `(config/kubernetes)`?

Reply:
Good call, done